docs/actions/capture_android_screenshots.md

```diff
@@ -319,14 +353,19 @@ Key | Description | Default
   `use_tests_in_packages` | Only run tests in these Java packages | 
   `use_tests_in_classes` | Only run tests in these Java classes | 
   `launch_arguments` | Additional launch arguments | 
-  `test_instrumentation_runner` | The fully qualified class name of your test instrumentation runner | `android.support.test.runner.AndroidJUnitRunner`
+  `test_instrumentation_runner` | The fully qualified class name of your test instrumentation runner | `androidx.test.runner.AndroidJUnitRunner`
   `ending_locale` | Return the device to this locale after running tests | `en-US`
+  `use_adb_root` | Restarts the adb daemon using `adb root` to allow access to screenshots directories on device. Use if getting 'Permission denied' errors | `false`
   `app_apk_path` | The path to the APK for the app under test | [*](#parameters-legend-dynamic)
   `tests_apk_path` | The path to the APK for the the tests bundle | [*](#parameters-legend-dynamic)
   `specific_device` | Use the device or emulator with the given serial number or qualifier | 
   `device_type` | Type of device used for screenshots. Matches Google Play Types (phone, sevenInch, tenInch, tv, wear) | `phone`
   `exit_on_test_failure` | Whether or not to exit Screengrab on test failure. Exiting on failure will not copy sceenshots to local machine nor open sceenshots summary | `true`
   `reinstall_app` | Enabling this option will automatically uninstall the application before running it | `false`
+  `use_timestamp_suffix` | Add timestamp suffix to screenshot filename | `true`
+  `adb_host` | Configure the host used by adb to connect, allows running on remote devices farm | 
+  `clean_status_bar` | Enabling this option will clean the status bar | `false`
+  `clean_status_bar_config` | Specifies the configuration for the clean status bar | `{}`
```

Comment:
@joshdholtz these options are no longer needed. I see that I forgot to remove them from my [PR](https://github.com/fastlane/fastlane/pull/15284/files#diff-f056ef18725e34d4c72839dd2ee839ca).

Reply:
Can you open a new PR that just removes them? Then this will be updated with the next release.

Reply:
@janpio  done. https://github.com/fastlane/fastlane/pull/15679